index.html

```diff
@@ -1,12 +1,476 @@
+                                    Report</a></button>
+                        </div>
+                    </div>
+                </div>
+               
+
+            </div>
+        </section>
+
+        <section class="two-cards section-margin">
+            <div class="container">
+                <div class="content">
+                    <div class="content-title">
+                        <h2>A New Approach to <a class="blue-text">Career Development</a></h2>
+                    </div>
+                
+
+                <div class="flex">
+
+                    <div class="column">
+
+                        <div class="column-img">
+                            <img src="img/illustration-ccms.svg" />
+                        </div>
+
+                        <div class="column-subtitle">
+                            <h3>Career Advantage Program</h3>
+                        </div>
+
+                        <div class="column-desc">
+                            <p>A half-day in-person program with a 45-day follow-up plan, the Career Advantage shows
+                                employees what it means to have meaningful career conversations, who they should be
+                                connecting with, and how engaging thoughtfully with the right people can lead to work
+                                with meaning.</p>
+                        </div>
+                        <div class="column-btn">
+                            <button class="button-learn-more"><a class="button-learn-more-text">Learn
+                                more</a></button>
+                        </div>
+                    </div>
+
+                    <div class="column">
+                        <div class="column-img">
+                            <img src="img/illustration-cb.svg" />
+                        </div>
+                        <div class="column-subtitle">
+                            <h3>Career Builder Program</h3>
+                        </div>
+                        <div class="column-desc">
+                           <p> Designed for high performers and future executives, the Career Builder program is a
+                            one-day tailored training program that allows employees to explore their strengths and
+                            career hopes, identify their ultimate goals, and chart a path within your organization
+                            to get there.
+                        </p>
+                        </div>
+                        <div class="column-btn">
+                            <button class="button-learn-more"><a class="button-learn-more-text">Learn
+                                more</a></button>
+                        </div>
+                    </div>
+                    </div>
+                </div>
+                
+            </div>
+        </section>
+
+        <section class="carousel">
```

Comment:
It's not just `carousel`. It's `testimonials`. Use suitable naming for each widget. It should correspond with the widget role.